src/main/scala/uk/gov/homeoffice/domain/core/email/EmailRepository.scala

```diff
@@ -67,6 +67,20 @@ trait EmailRepository extends Repository with MongoSupport with Logging {
     for { x <- emailCursor } yield Email(x)
   }
 
+  //Excluded html, text and cc from returned query as take up relatively large amount of space when running reports
+  def findEmailSummaryByDateRange(from: DateTime, to: Option[DateTime]): List[Email] = {
```

Comment:
All looks fine.
You may want to produce an iterable instead of a list if the query can potentially produce a large result e.g.

def findEmailSummaryByDateRange(from: DateTime, to: Option[DateTime]): Iterable[Email] = {
    val builder = MongoDBObject.empty

    val fields = MongoDBObject("user" -> 1, "emailId" -> 1, "caseId" -> 1, "caseRef" -> 1, "date" -> 1,
    "recipient" -> 1, "subject" -> 1, "status" -> 1, "emailType" -> 1)

    builder += Email.DATE -> dateRangeQuery(Some(from), to)

    collection.find(builder.result(), fields).toIterable.map(Email(_))
  }